web/src/javascript/leaderboardApp.js

```diff
@@ -11,7 +11,13 @@ function displayEntries(entries) {
   }
 }
 
-fetchEntries()
-  .then((entries) => {
-    displayEntries(entries);
-  });
+function fetchAndDisplay() {
+  return fetchEntries()
+    .then((entries) => {
+      displayEntries(entries);
+    });
+}
+
+setInterval(fetchAndDisplay, 30 * 1000);
```

Comment:
Personally I'd like a faster refresh (5s? 10s?), but it's not the end of the world if it's 30s.